dll/win32/msi/lang/msi_Id.rc

```diff
@@ -0,0 +1,179 @@
+/*
+ * English resources for MSI
```

Comment:
English resources? :thinking: Also, this module is actually winesynced, well however we already have own translations for it...

Reply:
i didn't look if msi module have own translation. will i keep this (my) translation? or continue with new own translation?

i will fix instead.

thanks, and sorry for late responds.

Reply:
It's entirely up to you. If you want to let the `msi_Id` translation as is and continue later we can merge the PR outright otherwise feel free to translate whatever missing parts there are. Make sure to mark your pull request as draft while doing so.

Reply:
>If you want to let the `msi_Id` translation as is and continue later we can merge the PR

For 'msi' translation, i think i'll let It be (so i can continue In another chance as part of other PR translation). So, i have done to do translation right now (this PR can be merged right now)